imports/plugins/core/checkout/client/components/completedOrder.js

```diff
@@ -18,95 +20,228 @@ import AddEmail from "./addEmail";
+                if (shipment.address.fullName || shipment.address.address1) {
+                  return (
+                    <div
+                      className="order-details-info-box"
+                      key={shipment._id}
+                    >
+                      <div
+                        className="order-details-info-box-content"
+                      >
+                        <p>
+                          {shipment.address.fullName}
+                          <br />
+                          {shipment.address.address1}
+                          <br />
+                          {shipment.address.city},
+                          {shipment.address.region}
+                          {shipment.address.postal}
+                          {shipment.address.country}
+                        </p>
+                      </div>
+                    </div>);
+                }
+              })
+              }
             </div>
-            {orderSummary.shipping.map((shipment) => {
-              if (shipment.address.fullName || shipment.address.address1) {
-                return <div className="order-details-info-box" key={shipment._id}>
-                  <div className="order-details-info-box-content">
-                    <p>
-                      {shipment.address.fullName}<br/>
-                      {shipment.address.address1}<br/>
-                      {shipment.address.city}, {shipment.address.region} {shipment.address.postal} {shipment.address.country}
-                    </p>
-                  </div>
-                </div>;
+
+            <div className="payment-info" >
+              <div className="order-details-content-title" >
+                <p>
+                  <Components.Translation
+                    defaultValue="Payment Method"
+                    i18nKey="cartCompleted.paymentMethod"
+                  />
+                </p >
+              </div>
+              {
+                this.state.paymentMethods.map(paymentMethod => (
+                  <CompletedOrderPaymentMethod
+                    key={paymentMethod.key}
+                    paymentMethod={paymentMethod}
+                  />))
               }
-            })}
+            </div>
           </div>
+          <CompletedOrderSummary
+            shops={
+              this.state.shops
+            }
+            orderSummary={
+              this.state.orderSummary
+            }
+            isProfilePage={
+              this.state.isProfilePage
+            }
+          />
+
+
+          { /* This is the right side / side content */}
+          <br />
+          <div className="row">
+            <div className="col-md-12">
+              {
 
-          <div className="payment-info">
-            <div className="order-details-content-title">
-              <p><Components.Translation defaultValue="Payment Method" i18nKey={"cartCompleted.paymentMethod"} /></p>
+                this.state.isCancelled ?
```

Comment:
If one value of the ternary operator is an empty string then you can write it as:
```js
{!this.state.isCancelled && (
  <CancelOrderButton
    isCancelled={this.state.isCancelled}
    cancelOrder={this.cancelOrder}
    order={this.state.order}
  />
)}